tests/functions.py

```diff
@@ -23,7 +23,7 @@
 authentication = (user, password)
 
 
-def GET(testpath, controller_a=False, **optional):
+def GET(testpath, payload={}, controller_a=False, **optional):
```

Comment:
Please don't specify mutable value as default arguments ( https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html )

Reply:
Ah, thanks didn't know about that, thanks